### ISSUE TITLE
couldnt run ppssp because of libzip.so.5 is not found

### ISSUE THREAD

**Issue by VergilGao** [spleen1981/retroarch-kodi-addon-CoreELEC#12]
when i run a game with ppsspp, it shows error:
```
[INFO] [Core]: Loading dynamic libretro core from: "/storage/.kodi/addons/script.retroarch.launcher.Amlogic-ng.arm/lib/libretro/ppsspp_libretro.so"
[ERROR] 打开libretro核心失败: "/storage/.kodi/addons/script.retroarch.launcher.Amlogic-ng.arm/lib/libretro/ppsspp_libretro.so"
Error(s): libzip.so.5: cannot open shared object file: No such file or directory
```
then i found no libzip isntalled:
```
CoreELEC:~/.kodi/addons/script.retroarch.launcher.Amlogic-ng.arm/lib # ldd libretro/ppsspp_libretro.so
ldd: $warning: you do not have execution permission for `libretro/ppsspp_libretro.so'
        /usr/lib/libarmmem-v7l.so (0xf6e85000)
        libpthread.so.0 => /usr/lib/libpthread.so.0 (0xf6e5c000)
        libzstd.so.1 => /storage/.kodi/addons/script.retroarch.launcher.Amlogic-ng.arm/lib/libzstd.so.1 (0xf6dc9000)
        libzip.so.5 => not found
        librt.so.1 => /usr/lib/librt.so.1 (0xf6db2000)
        libGLESv2.so => /usr/lib/libGLESv2.so (0xf530a000)
        libdl.so.2 => /usr/lib/libdl.so.2 (0xf52f7000)
        libavformat.so.58 => /usr/lib/libavformat.so.58 (0xf515f000)
        libavcodec.so.58 => /usr/lib/libavcodec.so.58 (0xf40e8000)
        libswresample.so.3 => /usr/lib/libswresample.so.3 (0xf40c4000)
        libswscale.so.5 => /usr/lib/libswscale.so.5 (0xf4052000)
        libavutil.so.56 => /usr/lib/libavutil.so.56 (0xf3dbb000)
        libz.so.1 => /usr/lib/libz.so.1 (0xf3da7000)
        libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0xf3c22000)
        libm.so.6 => /usr/lib/libm.so.6 (0xf3bb9000)
        libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0xf3b8c000)
        libc.so.6 => /usr/lib/libc.so.6 (0xf3a4d000)
        /usr/lib/ld-linux-armhf.so.3 (0xf778e000)
        libbz2.so.1.0 => /usr/lib/libbz2.so.1.0 (0xf3a2e000)
        libgnutls.so.30 => /usr/lib/libgnutls.so.30 (0xf3836000)
        libdav1d.so.5 => /usr/lib/libdav1d.so.5 (0xf374b000)
        libspeex.so.1 => /usr/lib/libspeex.so.1 (0xf3728000)
        libidn2.so.0 => /usr/lib/libidn2.so.0 (0xf36da000)
        libnettle.so.8 => /usr/lib/libnettle.so.8 (0xf3694000)
        libhogweed.so.6 => /usr/lib/libhogweed.so.6 (0xf3656000)
        libgmp.so.10 => /usr/lib/libgmp.so.10 (0xf35eb000)
```

**Comment by spleen1981** [comment]
fixed by 2377be72c6e80ec69128809edad3a75c5701b8f2

**Comment by VergilGao** [comment]
it works.
